### ISSUE TITLE
A lot of unexpected errors

### ISSUE THREAD

**Issue by JounQin** [azz/eslint-plugin-mdx#3]
```sh
in Slides on  master [+] ➜ yarn lint --fix                                                                      18:15:42 •99% 
yarn run v1.17.3
$ eslint . --ext mdx,ts,tsx

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide1.mdx
  1:10  error  'SubTitle' is defined but never used  @typescript-eslint/no-unused-vars
  1:40  error  Replace `'` with `⏎'⏎`                prettier/prettier

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide10.mdx
  2:63  error  Parsing error: ';' expected

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide11.mdx
  1:1   error  There should be at least one empty line between import groups  import/order
  1:8   error  'fs' is defined but never used                                 @typescript-eslint/no-unused-vars
  2:10  error  'ExternalLink' is defined but never used                       @typescript-eslint/no-unused-vars
  2:24  error  'SectionTitle' is defined but never used                       @typescript-eslint/no-unused-vars
  2:58  error  Replace `'` with `⏎'⏎`                                         prettier/prettier

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide2.mdx
  1:10  error  'SectionTitle' is defined but never used  @typescript-eslint/no-unused-vars
  1:44  error  Replace `'` with `⏎'⏎`                    prettier/prettier

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide3.mdx
  2:39  error  Parsing error: ';' expected

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide4.mdx
  2:53  error  Parsing error: ';' expected

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide5.mdx
  1:10  error  'SectionTitle' is defined but never used  @typescript-eslint/no-unused-vars
  1:44  error  Replace `'` with `⏎'⏎`                    prettier/prettier

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide6.mdx
  2:53  error  Parsing error: ';' expected

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide7.mdx
  2:53  error  Parsing error: ';' expected

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide8.mdx
  2:67  error  Parsing error: ';' expected

/Users/JounQin/Workspaces/GitHub/Slides/src/typescript/slide9.mdx
  2:53  error  Parsing error: ';' expected

✖ 18 problems (18 errors, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
📦 v0.1.0 via ⬢ v12.7.0 at ☸️  standard (alauda-system) took 10s
```

reproduction: https://github.com/JounQin/Slides/tree/eslint-plugin-mdx

**Comment by JounQin** [comment]
Close in favor of [@rxts/eslint-plugin-mdx](https://github.com/rx-ts/eslint-plugin-mdx).
